subprojects/internal-android-performance-testing/src/main/java/org/gradle/performance/android/SyncAction.java

```diff
@@ -34,10 +34,12 @@ public static void withProjectConnection(ProjectConnection connect, Action<? sup
         BuildActionExecuter<Map<String, AndroidProject>> modelBuilder = connect.action(new GetModel());
         modelBuilder.setStandardOutput(System.out);
         modelBuilder.setStandardError(System.err);
+        modelBuilder.forTasks("generateDebugSources");
         modelBuilder.withArguments("-Dcom.android.build.gradle.overrideVersionCheck=true",
             "-Pandroid.injected.build.model.only=true",
-            "-Pandroid.injected.invoked.from.ide=true",
```

Comment:
We had to remove this, because Android 3.0 no longer supports automatic sdk download when this property is set. It's not a big deal though, it only makes the Android plugin use machine-readable error messages instead of human-readable ones (which Android Studio can then parse). No other behavior changes.